modules/editorial-metadata/editorial-metadata.php

```diff
@@ -1395,7 +1395,7 @@ function print_configure_view() {
 		<?php if ( !isset( $_GET['action'] ) || ( isset( $_GET['action'] ) && $_GET['action'] != 'edit-term' ) ): ?>
 		<div id="col-right">
 		<div class="col-wrap">
-		<form id="posts-filter" action="" method="post">
+		<form action="" method="post">
```

Comment:
Two other modules `Custom Statuses` and `User Groups` do not have this element ID. So I think we may want to delete it and avoid confusion.

- http://github.com/Automattic/Edit-Flow/blob/c7e3fa7536b5787d494d54d3e34f240355e06dc6/modules/user-groups/user-groups.php#L596-L596
- http://github.com/Automattic/Edit-Flow/blob/c7e3fa7536b5787d494d54d3e34f240355e06dc6/modules/custom-status/custom-status.php#L1231-L1231